include/fuzzuf/utils/mmap_range.hpp

```diff
@@ -78,9 +78,9 @@ class mmap_iterator /*<T,
                         type_traits::RemoveCvrT<
                             typename std::iterator_traits<T>::value_type>,
                         fs::path>) {
-        cur = map_file((*base).string(), flags, populate);
+        cur = map_file((*base).string(), flags, populate, true);
```

Comment:
Add a comment on enabling `ignore_enoent` as it's a special case. 
```suggestion
        // NOTE: ignore ENOENT error because the file may be unlinked.
        cur = map_file((*base).string(), flags, populate, true);
```